experimental/go-dep/image/Dockerfile-stack

```diff
@@ -0,0 +1,35 @@
+FROM golang:1
```

Comment:
The README says this is based on `Go v1.12.5` however this image tag currently pulls `1.13.3`. 
I think you should at least lock this image to `golang:1.12`